app/policies/booking_policy.rb

```diff
@@ -0,0 +1,35 @@
+class BookingPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
```

Comment:
No. That means that the index will display all bookings in the database.
This is precisely why we need to limit the scope 
```suggestion
      scope.where(user: current_user)
```
not exactly sure of the format, so might need to check with TA

Reply:
or maybe we need to do this more comprehensive answer:

```
class Scope < Scope
    def initialize(user, scope)
      @user = user
      @scope = scope 
    end

    def resolve
      @scope.where(user = @user)
    end
end
```

Not really sure about this, but your version will definitely not work.
So again, probably show this to a TA to get a bulletproof version